.github/workflows/test.yml

```diff
@@ -253,7 +258,7 @@ jobs:
         run: |
           echo "PRISMA_CLIENT_ENGINE_TYPE=${{ matrix.queryEngine }}" >> $GITHUB_ENV
 
-      - run: docker-compose -f docker/docker-compose.yml up --detach postgres postgres_isolated mysql mysql_isolated mssql mongo cockroachdb
+      - run: docker-compose -f docker/docker-compose.yml up --detach postgres postgres_isolated mysql mysql_isolated mssql mongo cockroachdb vitess-8
```

Comment:
Is this needed?

Reply:
I don't think it's needed now because we have no tests in client functional tests that test on vitess. But as soon as we add one test, it will be needed.

Reply:
So @Jolg42, is the `relationMode` functional test suite excluded from the rest of the client functional tests unless `--relation-mode-tests-only` is specified? Or is it run regardless, but with the implicit `relationMode = "foreignKeys"` setting?

Reply:
`relationMode` tests are always ignored, unless `--relation-mode-tests-only` is passed, which we only do in the client relationMode job on GitHub Actions

Reply:
Removed.